.env

```diff
@@ -10,3 +13,4 @@ HMPPS_AUTH_BASE_URI=http://localhost:9090/auth
 HMPPS_AUTH_REDIRECT_BASE_URI=http://localhost:8080
 HMPPS_AUTH_CLIENT_ID=pecs-jpc-client
 HMPPS_AUTH_CLIENT_SECRET=clientsecret
+CRON_IMPORT_REPORTS=-
```

Comment:
@mafonso CRON disabled locally in the .env file.  Had real issues getting this to work with XARGS.  When I found something that worked without, this no longer worked with the Intelij plugin.